static/js/EN/EN.js

```diff
@@ -919,31 +920,37 @@ function renderAdaptiveResults(r) {
     } else if (r.acceptabilityII) {
         $('#sensationIII, #sensationII').html('Comfortable');
         if (to < r.tComfIUpper) {
-            $('#sensationI').html('Too cool');
+            // language=HTML
+            $('#sensationI').html('<span style="color:blue;">Too cool</span>');
         } else {
-            $('#sensationI').html('Too warm');
+            // language=HTML
+            $('#sensationI').html('<span style="color:red;">Too warm</span>');
         }
 	} else if (r.acceptabilityIII) {
         $('#sensationIII').html('Comfortable');
         if (to < r.tComfIIUpper) {
-            $('#sensationI, #sensationII').html('Too cool');
+            // language=HTML
+            $('#sensationI, #sensationII').html('<span style="color:blue;">Too cool</span>');
         } else {
-            $('#sensationI, #sensationII').html('Too warm');
+            // language=HTML
+            $('#sensationI, #sensationII').html('<span style="color:red;">Too warm</span>');
         }
     } else if (to < r.tComfIIIUpper) {
-        $('#sensationIII, #sensationII, #sensationI').html('Too cool');
+        // language=HTML
+        $('#sensationIII, #sensationII, #sensationI').html('<span style="color:blue;">Too cool</span>');
     } else {
-        $('#sensationIII, #sensationII, #sensationI').html('Too warm');
+        // language=HTML
+        $('#sensationIII, #sensationII, #sensationI').html('<span style="color:red;">Too warm</span>');
     }
 }
 
 function calcPmvCompliance(d, r) {
-    var pmv_complyI = Math.abs(r.pmv) <= 0.2;
-    var pmv_complyII = Math.abs(r.pmv) <= 0.5;
+    // var pmv_complyI = Math.abs(r.pmv) <= 0.2;
+    // var pmv_complyII = Math.abs(r.pmv) <= 0.5;
```

Comment:
Generally it's good to remove the commented code.. it makes things a bit confusing because it's hard to tell what's intentional

Reply:
Agree

Reply:
I left them there, since I do not want to change the code too much but those variables are not used.